scripts/run_benchmark.py

```diff
@@ -72,15 +36,20 @@ def bench_all():
         print(usage)
         exit(1)
     if sys.argv[1] == 'string':
-        bench_string()
+        data_types = ['string']
     elif sys.argv[1] == 'sorted':
-        bench_sorted()
+        data_types = ['sorted']
     elif sys.argv[1] == 'hash':
-        bench_hash()
+        data_types = ['hash']
     elif sys.argv[1] == 'queue':
-        bench_queue()
+        data_types = ['queue']
+    elif sys.argv[1] == 'blackhole':
+        data_types = ['blackhole']
     elif sys.argv[1] == 'all':
-        bench_all()
+        data_types = ['blackhole', 'string', 'sorted', 'hash', 'queue']
     else:
         print(usage)
         exit(1)
+    for [data_type, vsz_dist, k_dist] in itertools.product(data_types, value_size_distributions, key_distributions):
```

Comment:
Not all benchmarks supported by every data type. batch_write only supported by string, and range scan not supported by string.

Reply:
benchmark_impl.py is modified. Every test will skip unsupported data types.